src/app/booking/OOSPlannerSection.js

```diff
@@ -44,14 +44,50 @@ const BookingBox = styled(Box)(({ theme }) => ({
   padding: theme.spacing(2),
 }));
 
-function FilledBooking({ booking }) {
+function useBookingPolling({ bookingIdInitial, weekDates }) {
+  const { enqueueSnackbar } = useSnackbar();
+  const [counter, setCounter] = useState(0);
   const queryClient = useQueryClient();
+
+  const { data: newBookings } = useQuery(
+    ["newBookings", weekDates],
+    () => getBookings({ dateAfter: weekDates[0], dateBefore: weekDates[4] }), 
+    {
+      enabled: !!counter && counter <= 5,
```

Comment:
5 is a magic variable code smell.
https://refactoring.guru/replace-magic-number-with-symbolic-constant